k256/src/ecdsa/recoverable.rs

```diff
@@ -148,7 +151,7 @@ impl Signature {
     #[cfg_attr(docsrs, doc(cfg(feature = "ecdsa")))]
     #[cfg_attr(docsrs, doc(cfg(feature = "keccak256")))]
     pub fn recover_verify_key(&self, msg: &[u8]) -> Result<VerifyingKey> {
-        self.recover_verify_key_from_digest(Keccak256::new().chain(msg))
+        self.recover_verify_key_from_digest(Keccak256::new().chain_update(msg))
```

Comment:
I think you can use the following here:

```suggestion
        self.recover_verify_key_from_digest(Keccak256::new_with_prefix(msg))
```

Reply:
Done.